src/bootstrap/lib.rs

```diff
@@ -65,9 +65,15 @@
 //! also check out the `src/bootstrap/README.md` file for more information.
 
 #![deny(warnings)]
+#![feature(associated_consts)]
+#![feature(core_intrinsics)]
```

Comment:
So associated constants should be stabilized soon I believe -- there is a PR I think already?

The core intrinsics here is just used for sanity checking in the cache -- once we get non `'static` TypeId it should also be able to go away if we can serialize that, which should be possible.

I think it's fine to keep these for now, since the first is soon to be stabilized and the latter needs to be implemented.